templates/registration/password_reset_complete.html

```diff
@@ -6,9 +6,11 @@
 
 {% block content %}
 
-<div class="container">
-    <div class="d-flex flex-column">
-        <p class="m-auto">Your password has been reset. You may go ahead and <a href="{% url 'accounts:login_signup' %}">sign in</a> now.</p>
+<div class="container" style="margin-top: 150px;">
+    <div class="d-flex flex-column text-center" style="margin: auto; max-width: 500px; align-items: center;">
+        <i class="fas fa-check-circle my-3" style="color: #28a745; font-size: 7em;"></i>
+        <h2 class="font-weight-bolder my-3">Password Reset Successfully</h2>
+        <p class="m-auto"> You may go ahead and <a href="{% url 'accounts:login_signup' %}" style="color:#28a745">sign in</a> now.</p>
```

Comment:
For the link color, I think `#ff8c00` will look better as it goes with the theme.